src/nxdoc/nuxeo-server/administration/kafka.md

```diff
@@ -124,11 +135,120 @@ It is important to adapt the `max.poll.interval.ms` for slow consumers; otherwis
   ERROR [ComputationRunner] compliance: Exception in processLoop: Commit cannot be completed since the group has already rebalanced and assigned the partitions to another member. This means that the time between subsequent calls to poll() was longer than the configured max.poll.interval.ms, which typically implies that the poll loop is spending too much time message processing. You can address this either by increasing the session timeout or by reducing the maximum size of batches returned in poll() with max.poll.records.
   ```
 
-For instance, this will happen when using the `StreamWorkManager` if a Work takes more than 1h.
+For instance, this will happen when using the `StreamWorkManager` if a Work takes more than 2h.
 
 Please refer to the Kafka documentation about the [consumer and producer options](https://kafka.apache.org/documentation#configuration) and [replication](https://kafka.apache.org/documentation/#replication) for more information.
 
-When Kafka is used by the PubSub Provider, the **topic retention can be reduced to few hours** because PubSub is used to send instant messages, this can be done at the Kafka level using the following command:
+When using Nuxeo Stream for PubSub service (see below) it is recommended to **reduce the topic retention to few hours** in order to save disk storage.
```

Comment:
Why not setting the topic retention in the Platform by default?
Moreover should it be set to some minutes? Invalidations are pushed to the nodes and should be consumed very fast. And the DBS/VCS cache will probably be flushed before these few hours, so few hours seems to be too long

Reply:
Because retention duration is an admin configuration (like ACL), it is better managed at the Kafka level.
We could reduce the retention but 2 hours enables to see activity and debug without taking too much resource for the cleanup.
Also, pubsub is a generic service not specific to cache invalidation.